README.md

```diff
@@ -1,2 +1,13 @@
 # ansible-papertrail
-An ansible role for installing Papertrail
+An ansible role for installing [Papertrail](https://papertrailapp.com).
+
+## Role Variables
+
+- `papertrail_version` - Release of Papertrail's [remote_syslog2](https://github.com/papertrail/remote_syslog2) to use
+- `papertrail_check_interval_s` - Interval in seconds to check the logs (defaults to 10s)
+- `papertrail_port` - Must be set to user's Papertrail port
+- `papertrail_log_files` - List of logfiles to send; defaults to all .log files in /var/log
+
+## Example Playbook
+
```

Comment:
Should probably add a little extra note here similar to the one for our [collectd](https://github.com/azavea/ansible-collectd/#example-playbook) role explaining that `papertrail_port` _must_ be set to the users port and that by default it's going to be sending quite a bit of logging since it's sending everything in `/var/log/`